src/qibo/transpiler/blocks.py

```diff
@@ -95,13 +98,48 @@ def commute(self, block):
             return False
         return True
 
-    # TODO
-    def kak_decompose(self):  # pragma: no cover
-        """Return KAK decomposition of the block.
+    # Make a Circuit object and obtain the unitary by running the circuit
+    def unitary(self):
```

Comment:
```suggestion
    def _unitary(self):
```

I think the qiboteam usually prefers to denote with the underscore helper functions